addons/website_slides/static/src/xml/website_slides.xml

```diff
@@ -67,8 +67,9 @@
                     </main>
                     <footer class="modal-footer">
                         <button type="button" class="btn btn-secondary" data-dismiss="modal">Discard</button>
-                        <button type="button" data-loading-text="Loading..." class="btn btn-primary save">Save as Draft</button>
-                        <button type="button" data-loading-text="Loading..." data-published="true" class="btn btn-primary save">Save and Publish</button>
+                        <button type="button" data-loading-text="Loading..." t-if="widget.canUpload &amp;&amp; widget.canPublish" class="btn btn-primary save">Save as Draft</button>
```

Comment:
A little weird to have only "primary" buttons but it was like that before I guess...

Reply:
I like primary buttons.